pkgs/tools/text/mdbook-alerts/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, CoreServices
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-alerts";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "lambdalisue";
+    repo = "rs-mdbook-alerts";
+    rev = "v${version}";
+    hash = "sha256-LKNEI4dPXQwa+7JqLXpFZeKaQSSS5DFdeGuxEGNgPCU=";
+  };
+
+  cargoSha256 = "sha256-oycBTCjC58cO9q+eeO1nFGZGKdp6Bilgs8aFHW/4gXs=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [CoreServices];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
trcaking issue: #292468